tests/samples_test.go

```diff
@@ -140,6 +139,7 @@ func TestSimple(t *testing.T) {
 		testPath := path
 
 		t.Run(name, func(t *testing.T) {
+			t.Parallel()
```

Comment:
I never noticed that we weren't running the simple tests in parallel (or the JSON/Markdown tests). 🥴

Reply:
In CI:
```
$ go test -timeout 0 ./tests/...
ok  	github.com/chainguard-dev/malcontent/tests	391.531s
```
That shaved off about 2-3 minutes.